conf/php/php-fpm.conf

```diff
@@ -196,7 +196,7 @@ pm = static
 ; forget to tweak pm.* to fit your needs.
 ; Note: Used when pm is set to 'static', 'dynamic' or 'ondemand'
 ; Note: This value is mandatory.
-pm.max_children = 8
+pm.max_children = 3
```

Comment:
This was something i should have done in a branch to save memory on the my server that has many apps running on only 4 cores. I committed a revert of this for master.